Readme.md

```diff
@@ -397,7 +397,8 @@ program
   .addOption(new Option('-t, --timeout <delay>', 'timeout in seconds').default(60, 'one minute'))
   .addOption(new Option('-d, --drink <size>', 'drink size').choices(['small', 'medium', 'large']))
   .addOption(new Option('-p, --port <number>', 'port number').env('PORT'))
-  .addOption(new Option('--donate [amount]', 'optional donation in dollars').preset('20').argParser(parseFloat));
+  .addOption(new Option('--donate [amount]', 'optional donation in dollars').preset('20').argParser(parseFloat))
+  .addOption(new Option('-d, --disable-server', 'disables the server').conflicts(['port']));
```

Comment:
1) This new option will be displayed in the help, so example help needs updating.

2) Suggest add an example use generating an error in the examples below.

Reply:
1. Nice catch, done in https://github.com/tj/commander.js/pull/1678/commits/91ab6b11c0cababe6c6fbf3354693c126c6625fd

Reply:
2. Done in https://github.com/tj/commander.js/pull/1678/commits/79e6a7cec30712bf48bab0ad56d3ec2652ca7369
Also removed the shorthand version of `--disable-server` as it's already defined with `--drink`